exercises/csharp-2-a/.meta/test_messages.json

```diff
@@ -0,0 +1,32 @@
+{
+  "OneWordWithOneVowel": {
+    "cmd": "Sentence.WordWithMostVowels(\"a\")"
+  },
+  "OneWordWithOneVowelAndOneConsonant": {
+    "cmd": "Sentence.WordWithMostVowels(\"at\")"
+  },
+  "OneWordWithTwoVowelsAndOneConsonant": {
+    "cmd": "Sentence.WordWithMostVowels(\"tea\")"
+  },
+  "LongestWordIsNotWordWithMostVowels": {
+    "cmd": "Sentence.WordWithMostVowels(\"the area of a circle\")"
+  },
+  "LastWordIsWordWithMostVowels": {
+    "cmd": "Sentence.WordWithMostVowels(\"extremely fresh vegetables\")"
+  },
+  "MixedCasing": {
+    "cmd": "Sentence.WordWithMostVowels(\"TEST YOUR MIGHT\")"
```

Comment:
@iHiD Can you spot the pop-culture reference here?

Reply:
Nope! :)

Reply:
Apparently mortal compact. I never had a console with it on growing up!

Reply:
You didn't? Oh wow. It is indeed Mortal Kombat. I played that game so much 👊

Reply:
I wish I had.